pkg/client/create_builder_test.go

```diff
@@ -1079,11 +1081,11 @@ func testCreateBuilder(t *testing.T, when spec.G, it spec.S) {
 					h.AssertEq(t, len(layers), 1)
 				})
 
-				when("with exclude", func() {
+				when("only some modules are flattened", func() {
```

Comment:
Should this be nested under `flatten all`? I think it should be a sibling.

Reply:
Yeah..